services/server/youtube.ts

```diff
@@ -1,20 +1,24 @@
-type PlayListType = {
+export type YoutubeVideoLinkType = {
```

Comment:
```suggestion
export type VideoLink = {
```
make the type name more idiomatic for typescript

Reply:
done